### PR TITLE
Docs for orgs, rbac and sso

### PR DESCRIPTION
**Pull request by AlexejPenner**
## Describe changes
I described teh basics of orgs, rbac, and user authentication.

## Pre-requisites
Please ensure you have done the following:
- [ ] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/stacks-and-components/component-guide) table and the [corresponding website section](https://zenml.io/integrations).
- [ ] I have added tests to cover my changes.
- [ ] I have based my new branch on `develop` and the open PR is targeting `develop`. If your branch wasn't based on develop read [Contribution guide on rebasing branch to develop](https://github.com/zenml-io/zenml/blob/main/CONTRIBUTING.md#-pull-requests-rebase-your-branch-on-develop).

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

### CODE REVIEW

**Comment by AlexejPenner** [comment]
will redo, commit from gitbook changed too many files

File: docs/book/deploying-zenml/zenml-cloud/user-management.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+# User Management
+
+### Organizations, Tenants, and Roles
+
+ZenML Cloud arranges various aspects of your work experience around the concept
+of an **Organization.** This is the topmost level structure within the ZenML
```
Comment (htahir1) [inline docs/book/deploying-zenml/zenml-cloud/user-management.md:6]:
```suggestion
of an **Organization**. This is the top-most level structure within the ZenML
```

File: docs/book/deploying-zenml/zenml-cloud/user-management.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+Cloud environment. Generally, an organization contains a group of users and one
+or more tenants, which are individual, isolated deployments of the ZenML server.
```
Comment (htahir1) [inline docs/book/deploying-zenml/zenml-cloud/user-management.md:8]:
```suggestion
or more **tenants**. Tenants are individual, isolated deployments of the ZenML server.
```

File: docs/book/deploying-zenml/zenml-cloud/user-management.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+they can view, modify, and their level of involvement in collaborative tasks. A
+role thus helps determine the level of access that a user has within an
+organization.
+
+The `admin` has all permissions on an organization. They are allowed to add
+members, adjust the billing information and assign roles. The `editor` can still
+fully manage tenants and members but is not allowed to access the subscription
+information or delete the organization. The `viewer` Role allows you to allow
+users to access the tenants within the organization with no real permissions in
+the Cloud Control Plane.&#x20;
```
Comment (htahir1) [inline docs/book/deploying-zenml/zenml-cloud/user-management.md:20]:
```suggestion
users to access the tenants within the organization with only view permissions.
```

File: docs/book/deploying-zenml/zenml-cloud/user-management.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
+# User Management
+
+### Organizations, Tenants, and Roles
```
Comment (htahir1) [inline docs/book/deploying-zenml/zenml-cloud/user-management.md:3]:
```suggestion
## Organizations, Tenants, and Roles
```

File: docs/book/deploying-zenml/zenml-cloud/user-management.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+organization.
+
+The `admin` has all permissions on an organization. They are allowed to add
+members, adjust the billing information and assign roles. The `editor` can still
+fully manage tenants and members but is not allowed to access the subscription
+information or delete the organization. The `viewer` Role allows you to allow
+users to access the tenants within the organization with no real permissions in
+the Cloud Control Plane.&#x20;
+
+### Inviting Team Members
```
Comment (htahir1) [inline docs/book/deploying-zenml/zenml-cloud/user-management.md:22]:
```suggestion
## Inviting Team Members
```

File: docs/book/deploying-zenml/zenml-cloud/user-management.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+
+Inviting users to your organization to work on the organization's tenants is
+easy. Simply click `Add Member` in the Organization settings, and give them an
+initial Role. The User will be sent an invitation email. If a user is part of an
+organization, they can utilize their login on all tenants they have authority to
+access.
+
+### Using the ZenML CLI to connect to a Tenant
```
Comment (htahir1) [inline docs/book/deploying-zenml/zenml-cloud/user-management.md:30]:
```suggestion
## Using the ZenML CLI to connect to a Tenant
```

File: docs/book/deploying-zenml/zenml-cloud/user-management.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+
+To keep things simple, we can summarize the steps:
+
+1. Use the `zenml connect --url` command to start a device flow and connect to a
+   tenant.
+2. Choose whether to trust the device when prompted.
+3. Check permitted devices with `zenml devices list`.
+4. Invalidate a token with `zenml device lock ...`.
+
+#### Please remember:
```
Comment (htahir1) [inline docs/book/deploying-zenml/zenml-cloud/user-management.md:73]:
```suggestion
### Important notice
```

File: docs/book/deploying-zenml/zenml-cloud/user-management.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
+# User Management
+
```
Comment (htahir1) [inline docs/book/deploying-zenml/zenml-cloud/user-management.md:2]:
```suggestion

In ZenML Cloud, there is a slightly different entity heirarchy as compared to the open-source ZenML
framework. This document walks you through the key differences and new concepts that are cloud-only.
```

**Pull request merged**